addons/web/static/src/core/datepicker/datepicker.js

```diff
@@ -32,7 +32,7 @@ let datePickerId = 0;
  * @returns {boolean}
  */
 function isValidStaticFormat(format) {
-    return /^[\d\s/:-]+$/.test(DateTime.local().toFormat(format));
+    return /^[\d\s\.\/:-]+$/.test(DateTime.local().toFormat(format));
```

Comment:
Fix looks good.

cc @odoo/rd-spreadsheet => you have your own implementation of isValidStaticFormat which is essentially the same. You could use this one, what do you think ?

Reply:
We may want to allow other separators (`,`) ?

Reply:
@brboi yup we could definitely use it! I can't recall why we wrapped it in a try catch but anyway, we can def rely on this helper

Reply:
Thanks @rrahir 

@MerlinGuillaume could you make those changes (add the comma + deduplicate isValidStaticFormat) ?
Thanks ;-)

Reply:
@brboi Done :) Not sure where `isValidStaticFormat` should be located in the file